tests/conftest.py

```diff
@@ -283,17 +283,35 @@ def init_kwargs(request):
 
 
 @pytest.fixture(scope="function")
-def task(request, init_kwargs):
+def task(request, generate_task):
     """Gets a task instance."""
-    # set default values used for the initialization of the task.
-    init_args = {
-        "new_task_name": "",
-    }
+    new_task_name = _get_value_from_pytest_markers("new_task_name", request)
+    new_task_name = "" if new_task_name is None else new_task_name
```

Comment:
I think it's more Pythonic for `_get_value_from_pytest_markers` to take an optional `default=None` parameter and for the caller to pass "".

Also, swapping the request and marker_name would make it more like request.get(marker_name, default)

Reply:
Name suggestion: `_get_marker_value`

This is the only meaning of "marker" in this component, so there's no need to qualify it with "pytest".

Also, using "marker" to modify "value" is less wordy.

Reply:
request.get(marker_name, default) is an example of a dictionary-like API, not a literal suggestion for code to use.